book/types_of_data.md

```diff
@@ -336,24 +336,22 @@ To append one or more lists together, optionally with values interspersed in bet
 
 ## Tables
 
-The table is a core data structure in Nushell. As you run commands, you'll see that many of them return tables as output. A table has both rows and columns.
-
-We can create our own tables similarly to how we create a list. Because tables also contain columns and not just values, we pass in the name of the column values:
+A table is a list of records. Each record is a row and each key-value pair is a column.
 
 ```nu
-> [[column1, column2]; [Value1, Value2] [Value3, Value4]]
-╭───┬─────────┬─────────╮
-│ # │ column1 │ column2 │
-├───┼─────────┼─────────┤
-│ 0 │ Value1  │ Value2  │
-│ 1 │ Value3  │ Value4  │
-╰───┴─────────┴─────────╯
+> [{name: sam, rank: 10}, {name: bob, rank: 7}]
```

Comment:
I'll mention this in other commits too, but my personal preference is to not have commas in examples. It's really a superpower of nushell to not require commas and also not require quotes (this is a bare word).